src/lib/registration/ui/base_system_registration_dialog.rb

```diff
@@ -229,7 +229,19 @@ def default_url
       #
       # @return [String,nil] URL for the registration server; nil if not given.
       def boot_url
-        @boot_url ||= UrlHelpers.boot_reg_url
+        return nil unless UrlHelpers.boot_reg_url
+
+        unless valid_custom_url?(UrlHelpers.boot_reg_url)
+          Yast::Report.Error(
+            # TRANSLATORS: Wrong url for registration provided, %s is an URL.
+            _("The registration URL provided by Linuxrc is not valid.\n\n" \
```

Comment:
End users might not be familiar with `Linuxrc`, for them it's an URL written at the boot command line.